service-workers/service-worker/resources/fetch-event-respond-with-custom-response-worker.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+addEventListener('fetch', event => {
+  const url = new URL(event.request.url);
+  const type = url.searchParams.get('type');
```

Comment:
Does edge support search params now?  I guess we might use them in other SW tests already.